test/net/sourceforge/kolmafia/textui/RuntimeLibraryTest.java

```diff
@@ -1451,6 +1457,88 @@ public void itShouldMaximizeAndEquipSelectedWeapon() {
     }
   }
 
+  @Test
+  public void itShouldEquipCleaver() {
+    String maxStr =
+        "5item,meat,0.5initiative,0.1da 1000max,dr,0.5all res,1.5mainstat,-fumble,mox,0.4hp,0.2mp 1000max,3mp regen,0.25spell damage,1.75spell damage percent,2familiar weight,5familiar exp,10exp,5Mysticality experience percent,+200bonus spring shoes,+200bonus June cleaver,+200bonus designer sweatpants,2 dump";
+    HttpClientWrapper.setupFakeClient();
+    var cleanups =
+        new Cleanups(
+            withClass(AscensionClass.PASTAMANCER),
```

Comment:
It is possible that some of the withEquipped and withEquippableItem are redundant.  The withEquippableItem  should be deleted when this is recognized.